examples/StockDonor/src/StockDonorEditInput.tsx

```diff
@@ -25,27 +23,21 @@ const StyledInputRow = ({ label, Input }: InputWithLabelRowProps) => (
   />
 );
 
-const StockDonorEditInput = ({ data: stockLine }: StockDonorEditProps) => {
+const StockDonorEditInput: StockDonorEditPlugin = ({ stockLine, events }) => {
```

Comment:
Here the events are passed on as props, see docs PR for more explanation. I think this allows for more type safety, more flexibility and easier to understand